src/f4enix/output/fispact_legacy_out.py

```
@@ -271,3 +218,207 @@ def to_dataframe(self) -> pd.DataFrame:
+                line = lines[j + 1]
+                reactions = reaction_pat.findall(line)
+
+                if tokens[-2] in PATHWAY_END:
+                    # then there are no more tokens to parse
+                    break
+
+            elif "path continued" in line:
+                if zaids is None or reactions is None or perc is None:
+                    raise ValueError(
+                        f"Pathway continuation found but no pathway was being parsed. Text: {text}"
+                    )
```

Ruff (0.14.14)

[warning] Avoid specifying long messages outside the exception class

(TRY003)